scripts/build_mage_img.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+CURR_DIR="$PWD"
+MAGE_DIR="$CURR_DIR/mage"
+
+branch=$1
+image_name=$2
+deb_name=$3
+
+git clone --recurse-submodules -b $1 https://github.com/memgraph/mage.git
+
+cp "${CURR_DIR}/resources/memgraph-${deb_name}_amd64.deb" "${MAGE_DIR}/memgraph-${deb_name}_amd64.deb"
```

Comment:
I would at the beginning that MAGE_DIR is created if doesn't exist just to avoid possible errors.